net/hysteria/Makefile

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2023 Findlay Feng <i@fengch.me>
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=hysteria
+PKG_VERSION:=2.5.0
+PKG_RELEASE:=1
+
+GO_PKG:=github.com/apernet/hysteria
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://$(GO_PKG).git
+PKG_MIRROR_HASH:=98c4e776dc0c14de820a7113dc39a1cbe4c2ba07b203852bd2a7a65607f93fed
+PKG_SOURCE_VERSION:=app/v$(PKG_VERSION)
+
+PKG_LICENSE:=MIT
+PKG_MAINTAINER:=Findlay Feng <i@fengch.me>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(TOPDIR)/feeds/packages/lang/golang/golang-values.mk
+
+GO_PKG_BUILD_PKG=$(GO_PKG)/app/v2
+GO_PKG_LDFLAGS_X:= \
+	$(GO_PKG_BUILD_PKG)/cmd.appDate=$(shell date -u '+%F.%T') \
+	$(GO_PKG_BUILD_PKG)/cmd.appVersion=v$(PKG_VERSION) \
+	$(GO_PKG_BUILD_PKG)/cmd.appCommit=$(PKG_SOURCE_VERSION) \
+	$(GO_PKG_BUILD_PKG)/cmd.appPlatform=linux \
+	$(GO_PKG_BUILD_PKG)/cmd.appType=release \
+	$(GO_PKG_BUILD_PKG)/cmd.appArch=$(GO_ARCH)
+
+include $(GO_INCLUDE_DIR)/golang-package.mk
+
+define Package/$(PKG_NAME)
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Web Servers/Proxies
+	TITLE:=A feature-packed proxy & relay tool optimized for lossy
```

Comment:
optimised for lossy what?

Reply:
Use the headline from the website?

Reply:
Yes, I found his website updated the instructions